api.planx.uk/modules/analytics/docs.yaml

```diff
@@ -2,12 +2,27 @@ openapi: 3.1.0
 info:
   title: Plan✕ API
   version: 0.1.0
+
 tags:
   - name: analytics
+  - name: metabase
+
```

Comment:
Despite the plural "tags", Swagger only allows a route to be in one location - 

![image](https://github.com/user-attachments/assets/3cd62a94-0e24-4e03-a7e2-4000d234964f)

We generally want these to match the structure of out API modules - we can either being metabase out to it's own module, or keep it in analytics.

I'd argue it's it's own module (is there any shared types/code/logic?) but this isn't a blocker and we can make this decision later - it's just a case of moving file around.

For now, let's chose the easy path and just apply the analytics tag.